lib/carrierwave/storage/aws_file.rb

```diff
@@ -19,6 +19,12 @@ def file
         @file ||= bucket.object(path)
       end
 
+      def size
+        file.size
+      rescue Aws::S3::Errors::NotFound
+        nil
```

Comment:
I'm not sure that `nil` is the correct value to return when a file doesn't actually exist. Is that what other adapters return in the same situation?